SEO Optimization, KW "cloud continuous delivery" (#2824)
pulumi/docs: content/docs/guides/continuous-delivery/_index.md

---
title: Continuous Delivery for Cloud Resources

> meta_desc: Pulumi's approach to infrastructure as code is great for continuous delivery, because it uses source code to model cloud resources. Pulumi can easily integrate into any CI/CD system.

Reviewer: ```suggestion
meta_desc: Pulumi's approach to infrastructure as code is great for CI/CD because it uses source code to model cloud resources and integrates into any CI/CD system.
```

analysis tools, and running unit and integration tests as appropriate. It all "just works" for your cloud
infrastructure the same way it would for your application code.

> Pulumi can easily integrate into any continuous integration/continuous deliver (CI/CD) system. If your software isn't listed below or you are testing something new, see our guide for using Pulumi

Reviewer: ```suggestion
Pulumi can easily integrate into any continuous integration/continuous deliver (CI/CD) system. If CI/CD system isn't listed below or you are testing something new, see our guide for using Pulumi
```
